### PR TITLE
updated the geo lat,long with high accuracy codes

### PR DESCRIPTION
**Pull request by charanrajtc**
## ISSUE 

- Distance b/w zip 77059  to 77586 was showing 26 miles because of lat long precision.
actually it is around 5.33 miles https://www.searchbug.com/tools/zip-radius.aspx?TYPE=zipradius&ZIP=77059&DIST=20&submit=Search

## FIX
- updated the lat, long data form http://geonames.org to provide high accuracy distance computing. 

-----------------
Signed-off-by: Raj <charanraj.tc@gmail.com>

### CODE REVIEW

File: scripts/fetch.sh [hunk 1]
```diff
@@ -5,17 +5,29 @@ set -e
...
+    echo "Could not find unzip in path.."
+    exit 1;
+fi
+
 if [ ! -f ./free-zipcode-database.csv ]; then
     echo "Fetching US Zipcodes CSV File"
     $wget -nv "http://federalgovernmentzipcodes.us/free-zipcode-database.csv"
 fi
 
+if [ ! -f ./US.t ]; then
```
Comment (davglass) [inline scripts/fetch.sh]: Shouldn't this be:

```shell
if [ ! -f ./US.txt ]; then
```

File: scripts/process.js [hunk 2]
```diff
@@ -2,12 +2,13 @@
 
 var fs = require('fs'),
     path = require('path'),
-    zips = {}, str,
+    zips = {},
+    str,
     data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n');
+data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
```
Comment (davglass) [inline scripts/process.js]: This should be:

```js
    data = fs.readFileSync('./free-zipcode-database.csv', 'utf8').replace(/\r/g, '').split('\n'),
    data2 = fs.readFileSync('./US.txt', 'utf8').split('\n');
```

File: scripts/process.js [hunk 3]
```diff
@@ -55,4 +73,4 @@ for (var i in zips) {
 str = 'exports.codes = ' + JSON.stringify(zips) + ';\n';
 str += 'exports.stateMap = ' + JSON.stringify(stateMap) + ';\n';
 
-fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
+fs.writeFileSync(path.join('../', 'lib', 'codes.js'), str, 'utf8');
```
Comment (davglass) [inline scripts/process.js]: Please add a linefeed back here.

**Comment by charanrajtc** [comment]
done

Cheers,

{
  "name": "*Charan Raj T C*",
  "email": "charanraj.tc@gmail.com <mailto:charanraj.tc@gmail.com>",
  "telephone": "+91 96866 98989 <+919686698989>",
  "url": "http://coresoft.co.in/charanraj.tc"
}

**Comment by davglass** [comment]
Published in `zipcodes@5.0.0`

**Pull request merged**